templates/js/www

```diff
@@ -58,5 +58,5 @@ function onError(error) {
  */
 
 function onListening() {
-  debug('Listening on port ' + server.address().port);
+  debug('Listening on port ' + (server.address().port || server.address()));
```

Comment:
Yup, works perfect -- Dont know why the semicolon is missing at the end line..but otherwise looks good.